components/prism-pascaligo.js

```diff
@@ -0,0 +1,58 @@
+// Pascaligo is a layer 2 smart contract language for the tezos blockchain
+
+Prism.languages.pascaligo = {
+	'comment': [
+		/\(\*[\s\S]+?\*\)/,
+		/\/\/.*/
```

Comment:
I think it would better to combine these two patterns so that code like this `// (* foo *)` will be matched correctly.

So `/\(\*[\s\S]+?\*\)|\/\/.*/`